back-end/requests/post/generate_playlist.js

```diff
@@ -615,7 +621,65 @@ const add_songs_to_playlist = async (uris, user_id, bearer, generated_playlist_i
             JSON_RESPONSE=response.data
         })
         .catch((err) => {
-            const msg = "Something went wrong in the add_tracks endpoint"
+            const msg = "Something went wrong in the add_tracks axios"
+            console.log(msg)
+            console.log(uris)
+            console.error(err)
+            error = new Error(msg)
+    })
+
+    if (error)
+    {
+        return error
+    }
+
+    return JSON_RESPONSE
+}
+
+const remove_songs_in_playlist = async (user_id, bearer, generated_playlist_id) => {
+    const headers = {
+        'Content-Type': 'application/json',
+        'Authorization': `Bearer ${bearer}`
+    }
+
+    let GET_URL = `https://api.spotify.com/v1/playlists/${generated_playlist_id}/tracks`;
+    let JSON_RESPONSE
+    let uris
+    let error
+    await axios({
+        method: "get",
+        url: GET_URL,
+        headers: headers,
+      })
+        .then((response) => {
+            console.log("Successfully got tracks from playlist")
+            uris=response.data
```

Comment:
I believe this to be the problem with your code. response.data isn't actually a list of uri's, but it contains a lot of other information about the tracks as well. You'd have to sort of filter this out somehow to get just the track uris.